types/react-google-recaptcha/index.d.ts

```diff
@@ -79,6 +80,12 @@ export interface ReCAPTCHAProps {
 	 *  @see reCAPTCHA secure-token
 	 */
 	stoken?: string;
+	/**
+	 * Optional. Set the hl parameter, which allows the captcha to be used from different languages.
```

Comment:
Can we use the description defined by the official docs? https://developers.google.com/recaptcha/docs/display

which is

> Optional. Forces the widget to render in a specific language. Auto-detects the user's language if unspecified.

Reply:
Sure. I updated the description